src/server/report/app.js

```diff
@@ -141,24 +147,28 @@ class App extends LitElement {
 		`;
 	}
 	_goHome() {
-		page(this._root);
+		this._updateSearchParams({ file: undefined, test: undefined });
 	}
 	_handleFilterBrowserChange(e) {
-		const index = this._filterBrowsers.indexOf(e.target.value);
+		const browsers = [...this._filterBrowsers];
+		const index = browsers.indexOf(e.target.value);
 		if (!e.target.checked && index > -1) {
-			this._filterBrowsers.splice(index, 1);
+			browsers.splice(index, 1);
 		} else if (e.target.checked && index === -1) {
-			this._filterBrowsers.push(e.target.value);
+			browsers.push(e.target.value);
 		}
-		this._updateFiles();
+		this._updateSearchParams({ browsers: browsers.join(',') });
 	}
 	_handleFilterStatusChange(e) {
-		this._filterStatus = e.target.value;
-		this._updateFiles();
+		this._updateSearchParams({ status: e.target.value });
 	}
 	_handleSettingChange(e) {
 		this[`_${e.detail.name}`] = e.detail.value;
 	}
+	_handleTestClick(e) {
+		this._updateSearchParams({ file: e.target.dataset.file, test: e.target.dataset.test });
+		return false;
```

Comment:
Returning `false` from a click handler cases the `href` to not navigate. The `href` is still useful for opening links in new tabs.